docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -1,42 +1,66 @@
+Devfiles are YAML files, and they simplify your development process. In your IDE, devfiles tell you which best practices to apply to take your development project from source code to running application. Using the instructions given by devfiles, you can automate your current, manual processes. A devfile can take in the repository hosting your application source code. You can use the devfile to build your code, run your application on your local container, and then deploy your application to cloud-native containers.
 
-A devfile includes:
+.Features and benefits of devfiles
 
-* Guidance about runtime images
+Devfiles give guidelines and instructions for enhanced development processes. Devfiles use the command line interface (CLI)  tool, `odo`, to build run, and deploy your application. By using odo, devfiles provide consistent patterns of behavior for applications written in different languages and create workspaces composed of multiple containers. In these containers, you can create any number of identical workspaces from the same devfile. If you create multiple workspaces, you can share your devfile with different teams. By sharing a single devfile across multiple teams, you can ensure that each team has the same user experience and the same build, run, deploy behaviors.
+
+Using devfiles, you have the following features:
+
+* Guidance for using runtime images
 * Example code
 * Build and CI commands
 * Deployment options
 
-Benefits of devfiles:
+Use devfiles to help you with the following:
+
+* Reduce the gap between development and deployment
+* Set up a registry for finding and hosting stacks
+* Produce consistent build and run behaviors
+
+See the following example of a simple devfile:
+
+.A minimal devfile with one single project
+====
+[source,yaml]
+----
+schemaVersion: 2.1.0
+metadata:
+  name: petclinic-dev-environment
+  version: 1.0.0
+projects:
+  - name: petclinic
+    git:
+      remotes:
+        origin: "https://github.com/spring-projects/spring-petclinic.git"
+      checkoutFrom:
+        revision: main
+----
+====
+
+.Roles that use devfiles
 
-* Reducing the gap between development and deployment
-* Having a registry for finding and hosting devfiles
-* Producing consistent build and run behaviors
-* Allowing easy devfile customization
+Stack providers use devfiles to develop their runtime stacks once and then use these runtime stacks across different tools. Therefore, with devfiles, you do not need to build and maintain custom development tools.
 
-.Who can use devfiles?
+Tool providers use devfiles to build runtime support for each runtime. The devfile runtime teams build and run applications on their servers, giving you more time to work on your tools.
```

Comment:
Consider changing "The devfile runtime teams build and run applications on their servers, giving you more time to work on your tools." to "The devfile runtime teams create the devfiles that can be used to build and run applications on their servers, giving you more time to work on your tools."